client/homebrew/pages/userPage/userPage.jsx

```diff
@@ -24,7 +24,15 @@ const UserPage = createClass({
 	getDefaultProps : function() {
 		return {
 			username : '',
-			brews    : []
+			brews    : [],
+			sortType : 'alpha',
+			sortDir  : 'desc'
```

Comment:
Props represent any values that have been (or are expected to have been) "passed in" from a container object higher up (or from `Server.js`). At the moment, this page seems like it doesn't get passed those values in any way so we can just remove them here. Just tracking it in State is good enough.

Reply:
I'll pull those out in the next commit.